weasyl/middleware.py

```diff
@@ -132,30 +163,52 @@ def __call__(self, environ, start_response):
 
 class SentryEnvironmentMiddleware(object):
     def __init__(self, app, dsn, reactor=None):
-        import raven
         self.app = app
-        self.client = raven.Client(dsn)
+        self.client = raven.Client(
+            dsn=dsn,
+            release=raven.fetch_git_sha(os.path.abspath(os.path.join(__file__, '../..'))),
+            processors=[
+                'raven.processors.SanitizePasswordsProcessor',
```

Comment:
> So this was enough to catch the password confirmation fields too?

Yes.

> By the way, it's not clear to me whether the processors here are intended to make the things on sentry safe enough that sentry can be public-ish or if sentry should be locked down.

This is definitely not with the intent of making Sentry public. Session ids probably even still show up in locals somewhere.